cmd/cli/plugin/isolated-cluster/imageop/publishimagestotar.go

```diff
@@ -42,8 +42,7 @@ var PublishImagestotarCmd = &cobra.Command{
 }
 
 func init() {
-	PublishImagestotarCmd.Flags().StringVarP(&pullImage.TkgImageRepo, "source-repo", "", "projects.registry.vmware.com/tkg", "OCI repo where TKG bundles or images are hosted (required)")
-	_ = PublishImagestotarCmd.MarkFlagRequired("source-repo")
```

Comment:
I believe the `--source-repo` flag should not be required since it has a default value.  
If we want it to be required, we should remove the default value since it shows in help and is therefore confusing to the user (why have a default value but force me to specify my own value?).

If the goal is to require the flag but guide the user by showing the TKG repo URI, we should instead add it in the help text of the flag instead of as a default value.

Reply:
@Vandy-P Does this make sense?

Reply:
@marckhouzam  Yes correct , we should  remove require flag.